packages/ecc-utils-design/src/components/form/form.ts

```diff
@@ -548,9 +564,24 @@ export default class EccUtilsDesignForm extends LitElement {
       return this.renderArrayTemplate(field, newPath);
     }
 
-    if (field.fieldOptions?.required && !_.get(this.form, newPath)) {
-      this.requiredButEmpty.push(field.key);
+    if (field.fieldOptions?.required) {
+      if (
+        !_.get(this.form, newPath) &&
+        !this.requiredButEmpty.includes(field.key)
+      ) {
+        // add to requiredButEmpty
+
+        // eslint-disable-next-line no-empty
+        if (!this.hasUpdated && field.fieldOptions.default) {
+        } else this.requiredButEmpty.push(field.key);
+      } else if (_.get(this.form, newPath)) {
+        // remove from requiredButEmpty
+        this.requiredButEmpty = this.requiredButEmpty.filter(
+          (key) => key !== field.key
+        );
+      }
     }
+
```

Comment:
Is this supposed to be in this patch?

Reply:
yes, I made a had to fix a bug and figured i'd just add it to this patch. Or do you think it would be better to create another patch?